### ISSUE TITLE
Incorrect prediction

### ISSUE THREAD

**Issue by Abhijith002** [ProsusAI/finBERT#40]
Hi,

Used pre-trained models on financial news of a company for text classification. Given below are the three sentences which should have been predicted as positive. Is there a more refined model available or we need to further fine tune it ourselves ?  
![issue](https://user-images.githubusercontent.com/31363496/107410167-efc2f400-6b32-11eb-8fb3-6d7470c9b0f9.JPG)

Thanks,

**Comment by prats0599** [comment]
I'm facing the same issue. However the same model on this [link](https://huggingface.co/ProsusAI/finbert) seems to perform well. Did you find a solution?

**Comment by Abhijith002** [comment]
I did try to run the prediction using the model that is there on [huggingface](https://huggingface.co/ProsusAI/finbert), but it did not give me any better result. I did not find any solution so I am currently gathering the data for training.

**Comment by prats0599** [comment]
I tested the model on huggingface and it seems to be working fine now. All my predictions for the test set were negative which led me to believe something was wrong w my code. I finally got it to work with some tweaks to [the predict function in this repo here](https://github.com/ProsusAI/finBERT/blob/43f26c88561ff55412ceb86e2993ff92be938928/finbert/finbert.py#L580). I also used the Finbert Tokenizer("ProsusAI/finbert") instead of ''bert-base-uncased''.

**Comment by Abhijith002** [comment]
The prediction is going wrong particularly with statements l gave in the screenshot above. The statements like "Improving free cash flows has meant that the company's net debt also continues to decline" is a positive scenario for a company since there is decline in debt. But because of the word "decline" it is being predicted as negative.  There are 20 statements above and below those three which were predicted correctly.
![image](https://user-images.githubusercontent.com/31363496/109302395-3a48ae00-785f-11eb-8fd2-13da546a9a0f.png)

**Comment by doguaraci** [comment]
Hi, we don't have any other model. The model might make incorrect predictions. As reported on the [paper](https://github.com/ProsusAI/finBERT), the accuracy on the test set was 86%. 

With regards to @prats0599's problem of getting always negative predictions, that obviously shouldn't happen. @prats0599 could you please share more details about the problem and the fix you did?
